### PR TITLE
Adding conda unpack option to docker build / entrypoint

### PR DESCRIPTION
**Pull request by dtpryce**
Trying to remove as much logging as possible but allowing for development / debugging. I believe this should remove all the unpack logs from the pod start

### CODE REVIEW

File: hack/activate-env.sh [hunk 1]
```diff
@@ -40,7 +41,10 @@ _activateEnv() {
   set -u
 
   echo "--> Calling conda-unpack..."
-  conda-unpack
+  if [ $quiet ]; then
+    conda-unpack --quiet
```
Comment (adriangonz) [inline hack/activate-env.sh:45]: To keep things simple, should we always run `conda-pack` in quiet mode and remove the `_quiet` arg?

Comment (dtpryce) [reply hack/activate-env.sh:45]: True! Would that be ok for everyone? Does anyone use those logs?

Comment (dtpryce) [reply hack/activate-env.sh:45]: @adriangonz ^

Comment (adriangonz) [reply hack/activate-env.sh:45]: It should be alright to remove them :+1:

**Review by adriangonz — APPROVED** [review]
Nice one @dtpryce ! Looks great now!

**Comment by adriangonz** [comment]
Hey @dtpryce ,

It seems tests are failing due to an issue that has now been fixed in `master`. Would you be able to rebase?

**Comment by dtpryce** [comment]
> Hey @dtpryce ,
> 
> It seems tests are failing due to an issue that has now been fixed in `master`. Would you be able to rebase?

I think I have done it ... if this fails, I can make a new fork and re-request :D

**Comment by adriangonz** [comment]
Nice one! Looks all it's green! :rocket:

**Pull request merged**